spec/features/user/user_show_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe "the user show page", type: :feature do
+  context "before logging in" do
+    before  :each do 
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Trailing whitespace detected.